docs/guide/bulkinferrer.md

```diff
@@ -7,9 +7,9 @@ contains the original features and the prediction results.
 
 BulkInferrer consumes:
 
-*   A Trained model in [SavedModel](/guide/saved_model) format.
-*   Validation result from [Evaluator](evaluator) component.
-*   Unlabeled tf.Examples that contain features.
+*   A Trained model in [SavedModel](/guide/saved_model.md) format.
```

Comment:
Sorry did not catch this. `/guide/saved_model.md` seems to be non-existence.

@zhitaoli Do you happen to know where should this be pointed? Perhaps https://www.tensorflow.org/guide/saved_model ?

Reply:
Must've been changed, should I change all of them to https://www.tensorflow.org/guide as it might be more stable